app/controllers/catalog_controller.rb

```diff
@@ -41,6 +42,31 @@ def update_user_per_cat_page
     end
   end
 
+  def submit_cart_updates_form # rubocop:disable MethodLength, AbcSize
+    flash.clear
+    begin
+      # set the start and end date to updated values
+      # converts the dates into the expected format
+      start_date = Date.strptime(params[:form][:start_date], '%m/%d/%Y')
```

Comment:
This seems a bit excessive - is there no way to structure the forms identically so that we don't need to modify the `params` hash?